README.md

```diff
@@ -1173,6 +1197,83 @@ network_connections:
       key_mgmt: "owe"
 ```
 
+Examples of Applying the Network State Configuration
+-------------------
+
+Configuring the IP addresses:
+
+```yaml
+network_state:
+  interfaces:
+    - name: ethtest0
+      type: ethernet
+      state: up
+      ipv4:
+        address:
+          - ip: 192.168.122.250
+            prefix-length: 24
+        dhcp: false
+      ipv6:
+        address:
+          - ip: 2001:db8::1:1
+            prefix-length: 64
+        autoconf: false
+        dhcp: false
+    - name: ethtest1
+      type: ethernet
+      state: up
+      ipv4:
+        address:
+          - ip: 192.168.100.192
+            prefix-length: 24
+        auto-dns: false
+        dhcp: false
+      ipv6:
+        address:
+          - ip: 2001:db8::2:1
+            prefix-length: 64
+        autoconf: false
+        dhcp: false
```

Comment:
This is missing the `enabled` section.. in ipv4 and ipv6.

Reply:
Thanks. I updated.

Reply:
Thanks. I updated.